src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
+   conditioned on the site being variant). VCF genotype field reserved key GQ.
    */
-  union { null, float } mqRankSum = null;
+  union { null, int } quality = null;
 
+  // HQ Number=2 Type=Integer
   /**
-   The Wilcoxon rank-sum test statistic of the position of the base in the read at this site.
-   The positions are separated by whether or not the base supports the reference or the
-   alternate allele.
+   Haplotype quality, encoded as two phred qualities. VCF genotype field reserved key HQ, Number=2.
    */
-  union { null, float } readPositionRankSum = null;
+  array<int> haplotypeQuality = [];
 
+  // MQ Number=1 Type=Integer
   /**
-   The log scale prior probabilities of the various genotype states at this site.
-   The number of elements in this array should be equal to the ploidy at this
-   site, plus 1.
+   Root Mean Square (RMS) of the mapping quality of reads for this genotype. VCF genotype field
+   reserved key MQ.
    */
-  array<float> genotypePriors = [];
+  union { null, int } rmsMappingQuality = null;
 
+  // PL Number=G Type=Integer
   /**
-   The log scaled posterior probabilities of the various genotype states at this site,
-   in this sample. The number of elements in this array should be equal to the ploidy at
-   this site, plus 1.
+   Phred-scaled genotype likelihoods rounded to the closest integer, and otherwise defined precisely
+   as the genotype likelihoods field. VCF genotype field reserved key PL, Number=G.
    */
-  array<float> genotypePosteriors = [];
+  array<int> phredScaledLikelihoods = [];
 
+  // PQ Number=1 Type=Integer
   /**
-   The log-odds ratio of being a true vs. false variant under a trained statistical model.
-   This model can be a multivariate Gaussian mixture, support vector machine, etc.
+   Phred-scaled probability that alleles are ordered incorrectly, against all other members in
+   the same phase set. VCF genotype field reserved key PQ.
    */
-  union { null, float } vqslod = null;
+  union { null, int } phaseSetQuality = null;
 
-  /**
-   If known, the feature that contributed the most to this variant being classified as
-   a false variant.
-   */
-  union { null, string } culprit = null;
 
+  // SB is not a reserved key in VCF v4.2 or v4.3
   /**
-   Additional feature info that doesn't fit into the standard fields above.
-   They are all encoded as (string, string) key-value pairs.
+   Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias.
+   If set, the array should contain the number of reads that support the reference allele on the
+   forward strand, the reference allele on the reverse strand, the alternate allele on the forward
+   strand, and the alternate allele on the reverse strand, in that order. VCF genotype field key SB.
    */
-  map<string> attributes = {};
-}
+  //array<int> strandBias = [];  Genotype.strandBiasComponents previously
```

Comment:
Actually, ignore what I was saying earlier. Since you've added ADF/ADR, I would prefer to use the ADF/ADR fields over using SB.